src/gmail.js

```diff
@@ -982,7 +982,7 @@ var Gmail = function(localJQuery) {
     api.check.data.is_email_id = function(id) {
         return id
             && typeof id === "string"
-            && /^msg-a:/.test(id);
+            && /^msg-[a|f]:/.test(id);
     };
```

Comment:
These are good improvements and obviously something I missed first time around. Thanks for fixing!